README.rst

```diff
@@ -10,3 +10,11 @@ mergify-engine
     :alt: Build Status
 
 Stop merging your pull requests manually, more info on https://mergify.io
+
+Development Setup
+=================
+
+pip install .
+set fake.env variables (Windows: set VARIABLE=XXX)
+SET FLASK_APP=mergify_engine/web.py
+python -m flask run
```

Comment:
I'd prefer we focus on the goal of the PR, not sure those instructions are actually the "state of the art" :)